qemu/tests/migration_with_sgx.py

```diff
@@ -0,0 +1,52 @@
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+from provider.sgx import SGXHostCapability
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Do migration test with sgx enabled
+
+    1. Check sgx and numa capability
+    2. Boot src and dst sgx guests with numa
+    3. Migration
+    4. Check dst VM sgx enabled in guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    error_context.context("Start sgx test", test.log.info)
+    timeout = float(params.get_numeric("login_timeout", 240))
```

Comment:
we don't need to use a float timeout value for wait_for_login function

Reply:
Agree with Zhenchao.

Reply:
Updated thanks